src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -103,22 +103,33 @@ class CrossDeviceLinkUI extends Component {
       copySuccess: false,
       sending: false,
       error: {},
-      validNumber: true,
+      validNumber: true
     }
   }
 
   linkId = `${process.env.BASE_32_VERSION}${this.props.roomId}`
 
   linkCopiedTimeoutId = null
 
-  copyToClipboard = (e) => {
-    this.linkText.select()
-    document.execCommand('copy')
-    e.target.focus()
+  copyToClipboard = (mobileUrl) => {
```

Comment:
[Minor] This can be decoupled from the component as an utility that executes a callback, IMO.